Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -296,7 +296,9 @@ inline ListStyleType BuilderConverter::convertListStyleType(const BuilderState&
 {
     auto& primitiveValue = downcast<CSSPrimitiveValue>(value);
     if (primitiveValue.isValueID()) {
-        if (builderState.document().settings().cssCounterStyleAtRulesEnabled() && !isCounterStyleUnsupportedByUserAgent(primitiveValue.valueID()))
+        if (primitiveValue.valueID() == CSSValueNone)
```

Comment:
`fromCSSValue()` should already handle `CSSValueNone`. Are you doing this explicitly as an optimization for a frequent value?

Reply:
Mostly separating this out to clearly distinguish old and new codepath